File: spec/code_samples/Perl_perl5/boards@hidden_entries/patch.pl
#!/usr/bin/perl
use Modern::Perl;
use Data::Dumper;
use HTTP::Request::JSON;
use LWP::UserAgent::JSON;
my $request = HTTP::Request::JSON->new("PATCH",
"https://www.directfreight.com/api/boards/hidden_entries");
$request->content('{"entry_id":["<ADD STRING VALUE>"]}');
my $browser = LWP::UserAgent::JSON->new();
my $response = $browser->request($request);
print Dumper($response->json_content);

File: spec/code_samples/Php_http1/boards@hidden_entries/patch.php
<?php

HttpRequest::methodRegister('PATCH');
$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/boards/hidden_entries');
$request->setMethod(HttpRequest::HTTP_METH_PATCH);

$request->setBody('{"entry_id":["<ADD STRING VALUE>"]}');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}